README.md

```diff
@@ -13,7 +13,7 @@ Laravel HTMLMin is currently maintained by [Raza Mehdi](https://github.com/srmkl
 
 ## Installation
 
-Laravel HTMLMin requires [PHP](https://php.net) 5.5+. This particular version supports Laravel 5.1, 5.2, 5.3, or 5.4 only.
+Laravel HTMLMin requires [PHP](https://php.net) 5.5+, and supports Laravel 5.1, 5.2, 5.3, 5.4, and 5.5 only.
```

Comment:
Please keep the text the same. This particular version is correct, not any version,